### DISCUSSION TITLE
各位大佬 我想问下 你们有遇到过 antd: 5.2.2 版本下 无法加载的原因吗 package.json如下

### DISCUSSION THREAD

**Discussion by tjt740** [alibaba/formily#3864] [Q&A 2.x]
{
  "name": "ainlp",
  "description": "",
  "version": "0.1.0",
  "keyword": [],
  "author": "Cone - Cainiao Inc,Alibaba group.",
  "repository": {
    "type": "git",
    "url": "git+https://gitlab.alibaba-inc.com/nbfalcon/ainlp.git"
  },
  "engines": {
    "node": ">=14.0.0"
  },
  "scripts": {
    "dev": "tnpm run start",
    "start": "cone start --port 7200",
    "build": "cone build",
    "start-storybook": "cone start-storybook -p 7300",
    "build-storybook": "cone build-storybook",
    "test": "cone test",
    "lint": "cone lint",
    "format": "cone format",
    "prepare": "husky install",
    "pre-commit": "lint-staged",
    "commit-msg": "commitlint --edit"
  },
  "lint-staged": {
    "*.{json,md}": [
      "prettier --write --ignore-unknown"
    ],
    "*.{js,jsx,ts,tsx}": [
      "eslint",
      "prettier --write --ignore-unknown"
    ],
    "*.{css,scss,less}": [
      "stylelint",
      "prettier --write --ignore-unknown"
    ]
  },
  "commitlint": {
    "extends": [
      "@commitlint/config-conventional"
    ]
  },
  "dependencies": {
    "@ali/global-locale": "^1.0.5",
    "@ali/global-string-format": "^1.0.7",
    "@alifd/next": "^1.26.9",
    "@alife/cainiao-error-boundary": "^0.2.0",
    "@alife/cn-env": "^0.1.0",
    "@alife/cn-request": "^1.0.2",
    "@alife/cn-standard-layout": "^0.2.0",
    **"@alife/cn-ui": "0.9.39",**
    "@alife/cone-arms": "^0.1.0",
    "@alife/cone-config-provider": "^0.1.0",
    "@alife/mcms_cone-market_cone": "^0.0.26",
    "@alife/micro-stark": "^0.2.2",
    "@alife/panda-i18n": "^0.2.0",
    "@alife/theme-26075": "^1.3.5",
    "@alife/web-component-sample": "^0.1.0",
    "@ant-design/icons": "^5.0.0",
    "@ice/router": "^0.1.1",
    "@monaco-editor/react": "^4.5.1",
    "@rematch/core": "^2.1.0",
    "@rematch/immer": "^2.1.0",
    "@rematch/loading": "^2.1.0",
    "@rematch/select": "^3.1.0",
    "@svgdotjs/svg.draggable.js": "^3.0.2",
    "@svgdotjs/svg.js": "^3.1.2",
    "@svgdotjs/svg.panzoom.js": "^2.1.2",
    "@types/marked": "^5.0.0",
    "ace-builds": "^1.20.0",
    "ali-oss": "^6.17.1",
    **"antd": "^5.2.2",**
    "brace": "^0.11.1",
    "buc-client": "^3.12.6",
    "canvas-select": "^2.9.1",
    "classnames": "^2.3.1",
    "dayjs": "^1.11.6",
    "github-markdown-css": "^5.2.0",
    "gsap": "^3.11.5",
    "highlight.js": "^11.8.0",
    "immer": "^9.0.6",
    "lodash": "^4.17.21",
    "marked": "^5.0.2",
    "marked-highlight": "^2.0.0",
    "moment": "^2.29.4",
    "normalize.css": "^8.0.1",
    "prismjs": "^1.29.0",
    "prop-types": "^15.7.2",
    "react": "^17.0.2",
    "react-ace": "^10.1.0",
    "react-color": "^2.19.3",
    "react-dom": "^17.0.2",
    "react-json-view": "^1.21.3",
    "react-redux": "^7.1.0",
    "react-use": "^17.3.1",
    "svg.draggable.js": "2.2.2",
    "svg.draw.js": "^2.0.4",
    "svg.js": "2.7.1",
    "svg.panzoom.js": "^1.2.3",
    "svg.resize.js": "^1.4.3",
    "svg.select.js": "^3.0.1"
  },
  "devDependencies": {
    "@ali/build-plugin-mjs-adapter": "^1.0.0",
    "@alicd/tailwind-config-cone": "^0.2.7",
    "@alife/cone-cli": "^0.3.83",
    "@alife/eslint-config-cone": "^0.2.0",
    "@alife/jest-config-cone": "^0.1.0",
    "@alife/prettier-config-cone": "^0.1.0",
    "@alife/storybook-config-cone": "^0.4",
    "@alife/stylelint-config-cone": "^0.2.0",
    "@alife/ts-config-cone": "^0.1.0",
    "@commitlint/cli": "^14.1.0",
    "@commitlint/config-conventional": "^14.1.0",
    "@storybook/addon-docs": "^6.3.2",
    "@storybook/react": "^6.3.12",
    "@types/jest": "^26.0.2",
    "@types/lodash": "^4.14.168",
    "@types/react": "^17.0.3",
    "@types/react-dom": "^17.0.9",
    "build-plugin-cone-analyse": "^0.1.0",
    "build-plugin-cone-app": "^0.2.97",
    "build-plugin-fusion": "^0.1.2",
    "build-plugin-jsx-plus": "^0.1.1",
    "build-plugin-moment-locales": "^0.1.0",
    "build-plugin-multi-pages": "^0.1.3",
    "build-plugin-react-app": "^2.2.3",
    "build-plugin-router": "^1.x",
    "eslint": "^8.2.0",
    "eslint-webpack-plugin": "^3.2.0",
    "husky": "^7.0.2",
    "jest": "^27.2.2",
    "lint-staged": "^12.0.2",
    "prettier": "^2.2.1",
    "sass": "^1.43.2",
    "stylelint": "^14.0.1",
    "type-fest": "^2.5.3",
    "typescript": "^4.4.4",
    "typescript-plugin-css-modules": "^3.4.0"
  },
  "resolutions": {
    "webpack": "^4"
  }
}

**Comment by HAjiMa666** [comment]
貌似不支持，formily支持antd<=4.22.8版本的
